### DISCUSSION TITLE
What feature ideas have you got?

### DISCUSSION THREAD

**Discussion by dcallus** [fillmaster/fillMaster#38] [Ideas]
Let me know.
 
 There is lots in the pipeline, but let us know what ideas you have. However small or big.

**Comment by kristersd** [comment]
I can help with ideas for UI or implementation, code specific, as I don't have an any experience being an musician or drummer.

**Reply by dcallus** [reply]
I've been sitting at the computer all day, so going to go and take a walk outside and get the remaining sun with the doggo. But I'll be in touch re: this soon.

I'm currently sketching up some ideas for UI improvements based on some advice from some UI folks, so I'll run them past you as soon as I can and we can work through some stuff together.

I'm not sure if you can DM on here, so if you go to the reddit post I made about this app (assuming that's how you got here) you can send me a DM on there if you want with your email address and I will add you to the slack channel.

I am aware slack is not (according to some) the best way to communicate for open source stuff, but it will do for now until we have something else.
